src/screens/AccountScreens/ProfileScreen/style.js

```diff
@@ -7,5 +7,31 @@ export const styles = StyleSheet.create({
     flex: 1,
     backgroundColor: THEME_COLORS.WHITE,
     paddingTop: '15%',
+    alignItems: "center",
   },
+  image: {
+    marginTop: "15%",
+    width: "60%",
+    height: "30%",
+  },
+  userInfoContainer: {
+    marginTop: "5%",
+    width: "60%",
+    height: "30%",
+    justifyContent: "center",
+  },
+  userInfoTextContainer: {
+    height: "20%",
+    width: "100%",
+    borderBottomColor: "#ddd",
+    borderBottomWidth: 2,
+    alignItems: "center",
+    marginBottom: "10%",
+  },
+  userInfoText: {
+    fontSize: 24,
```

Comment:
We can create an object in `src/components/theme.js` named as `SYSTEM_FONT` or something similar for the text elements. We can treat this object as an Enum and easily reuse it in the future. In this way, styling can be more manageable, and it can ensure the consistency of text elements across the app. It's also easier to understand in terms of code readability. You can follow the implementation of `THEME_COLORS`.